sem-reference_5b6968642c7d3a03f89d6bff.md

```diff
@@ -924,6 +925,61 @@ spec:
 When you save the changes and leave the editor, your changes will be applied to
 the project.
 
+### Changing Github status notifications
+
+By default, Semaphore will create status checks only for initial pipeline.
+
+To modify this, edit the status property in repository section in the project
+YAML.
+
+For example, to modify the granularity of statuses from pipeline to block level
+edit your project with the following command:
+
+``` bash
+sem edit project example-project-1
+```
+
+In the editor, modify the `status/pipeline_files` entry:
+
+``` yaml
+# Editing Projects/example-project-1.
+
+apiVersion: v1alpha
+kind: Project
+metadata:
+  name: example-project-1
+  id: 37db0a22-592c-45e2-bdeb-799ba977502c
+  description: "Example project"
+spec:
+  repository:
+    url: git@github.com:example/project-1.git
+    run_on:
+      - tags
+      - branches
+    pipeline_file: ".semaphore/semaphore.yml"
+    status:
+      pipeline_files:
+      - path: ".semaphore/semaphore.yml"
+        #
+        # Edit this line to set a new level of granularity
+        #
+        level: "block"
+```
+
+When you save the changes and leave the editor, your changes will be applied to
+the project.
+
+To add statuses from promoted pipelines modify the `pipeline_files` entry and
```

Comment:
There should be a subheader above to clearly divide this as a different example.